RapidReact/src/main/java/frc/robot/subsystem/DrivetrainSubsystem.java

```diff
@@ -26,80 +26,91 @@
    * <p>
    * This can be reduced to cap the robot's maximum speed. Typically, this is useful during initial testing of the robot.
    */
-  public static final double MAX_VOLTAGE = 12.0;
-  // FIXME Measure the drivetrain's maximum velocity or calculate the theoretical.
+  public final double maxVoltage = 12.0;
```

Comment:
Can we move this to config? More than one subsystem are going to want maxVoltage.